### ISSUE TITLE
Test for WCAG technique G78: Providing a second, user-selectable, audio track that includes audio descriptions

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#197]
http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G78
The objective of this technique is to provide an audio (spoken) version of information that is provided visually so that it is possible for people who cannot see to be able to understand audio-visual material.

Since most user agents today cannot merge multiple sound tracks, this technique adds the additional audio information to synchronized media by providing an option which allows users to replace the soundtrack with a new copy of the original soundtrack that has the additional audio description added. This added information focuses on actions, characters, scene changes and on-screen text (not captions) that are important to understanding the content.

Since it is not helpful to have this new information obscure key audio information in the original sound track (or be obscured by loud sound effects), the new information is added during pauses in dialogue and sound effects. This limits the amount of supplementary information that can be added to the program.

The soundtrack with the audio description (of visual information) can either be an alternate sound track that the user can choose, or it can be the standard sound track that everyone hears.
## Test procedure
1. Check that the audio track that includes audio descriptions can be turned on by using a control within the content itself or by selecting a preference in the media player.
2. Listen to the synchronized media
3. Check to see if gaps in dialogue are used to convey important information regarding visual content
## QUAIL

We can check if a .mp3 is included in the video element or object (same way as for captions in #93 ). Probably the outcome of this test should be pass or cantTell as we can't detect wheter or not the normal audio track gives enough information.
